packages/blade/.storybook/react/preview.tsx

```diff
@@ -46,6 +48,42 @@ export const parameters = {
           'overrideTheme',
         ],
         'Components',
+        [
+          'Button',
```

Comment:
@saurabhdaware lets resume [this](https://github.com/razorpay/blade/pull/1484#discussion_r1280246321) discussion here